pom.xml

```
@@ -252,7 +252,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-2.361.x</artifactId>
```

```suggestion
        <artifactId>bom-2.387.x</artifactId>
```
to match `jenkins.version`